README.md

```diff
@@ -1 +1,33 @@
-# testing-appserver
+# ONLYOFFICE AppServer testing suit
```

Comment:
Also I think it can be name `ONLYOFFICE AppServer and ONLYOFFICE Personal testing suite`